Sources/Views/InputBarItem.swift

```diff
@@ -36,24 +47,25 @@ open class InputBarButtonItem: UIButton {
     
     // MARK: - Properties
     
-    /// Set automatically when using the InputBarAccessoryView method 'setStackViewItems'
+    /// A weak reference to the MessageInputBar that the InputBarButtonItem used in
     open weak var messageInputBar: MessageInputBar?
     
-    public var spacing: Spacing = .none {
+    /// The spacing property of the InputBarButtonItem
+    open var spacing: Spacing = .none {
```

Comment:
Think we can make this more descriptive?

Reply:
/// The spacing property of the InputBarButtonItem that determines the contentHuggingPriority and any
    /// additional space to the intrinsicContentSize